astropy/cosmology/flrw/base.py

```diff
@@ -178,7 +178,7 @@ class FLRW(Cosmology, _ScaleFactorMixin):
     """
 
     H0: Parameter = Parameter(
-        doc="Hubble constant as an `~astropy.units.Quantity` at z=0.",
+        doc="Hubble constant as an |Quantity| at z=0.",
```

Comment:
an -> a

Actually, maybe better to just delete "as a |Quantity|", it is a bit superfluous and you don't have it for hubble time, and removed it in quite a few other places.

Reply:
Agreed about removal.